### PR TITLE
minor spelling tweaks in markdown files under g3doc directory

### PR DESCRIPTION
**Pull request by kiszk**

### CODE REVIEW

File: g3doc/ConversionToLLVMDialect.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ type will be a wrapped LLVM IR structure type where each element of the
 structure corresponds to one of the results of the original function, converted
 using these rules. In high-order functions, function-typed arguments and results
 are converted to a wrapped LLVM IR function pointer type (since LLVM IR does not
-allow passing functions to functions without indirection) with the pointee type
+allow passing functions to functions without indirection) with the pointer type
```
Comment (joker-eph) [inline g3doc/ConversionToLLVMDialect.md]: I think pointee was intentional here.

File: g3doc/OpDefinitions.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ best practices. They are rendered as quoted bullet points.
 ## Motivation
 
 MLIR allows pluggable dialects, and dialects contain, among others, a list of
-operations. This open and extensible ecosystem leads to the "stringly" type IR
+operations. This open and extensible ecosystem leads to the "strongly" type IR
```
Comment (joker-eph) [inline g3doc/OpDefinitions.md]: stringly was intended here.

**Comment by joker-eph** [comment]
Thanks for the contributions!

File: g3doc/ConversionToLLVMDialect.md (hunk 1 above)
Comment (kiszk) [reply g3doc/ConversionToLLVMDialect.md]: Sure, done

File: g3doc/OpDefinitions.md (hunk 2 above)
Comment (kiszk) [reply g3doc/OpDefinitions.md]: Thank you for pointing out. I should have read the next line carefully.

**Pull request closed without merging**